prow/cmd/deck/static/script.js

```diff
@@ -26,16 +26,16 @@ function optionsForRepo(repo) {
         states: {},
     };
 
-    for (var i = 0; i < allBuilds.length; i++) {
-        var build = allBuilds[i];
-        opts.types[build.type] = true;
-        opts.repos[build.repo] = true;
-        if (!repo || repo === build.repo) {
-            opts.jobs[build.job] = true;
-            if (build.type === "presubmit") {
-                opts.authors[build.author] = true;
-                opts.pulls[build.number] = true;
-                opts.states[build.state] = true;
+    for (var i = 0; i < allBuilds.items.length; i++) {
+        var build = allBuilds.items[i];
+        opts.types[build.spec.type] = true;
+        opts.repos[build.spec.refs.repo] = true;
+        if (!repo || repo === build.spec.refs.repo) {
+            opts.jobs[build.spec.job] = true;
+            if (build.spec.type === "presubmit") {
+                opts.authors[build.spec.refs.pulls[0]['author']] = true;
```

Comment:
Do we want to not squash the list of pulls for batches?

Reply:
is there any case that build.spec.refs.pulls can be `null` or `undefined`?

Reply:
Yes, in periodic and postsubmit jobs there will be no pulls